package.json

```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "devDependencies": {
     "tslint": "^4.4.2",
-    "typescript": "^2.3.4"
+    "typescript": "3.2.4"
```

Comment:
One question about peerDeps:
1. Do we really need `typescript` there? As I go through the history of this repo I have a feeling that we needed it in the time when we had our custom rule. Now, the custom rule is deleted so I just can not find any reason for having ts as a peer dep.
2. However, if we need to have `ts` as a peer dep then shouldn't we bump its version to be  `^3.2.4` ?

Reply:
One of the goals of this package is to enforce a consistent TS version across the projects that use it. In this case, we're specifically locking to 3.2.4 because that's the highest version that is compatible with Angular 7.

Reply:
Oh yeah, that is correct - I thought we were looking at peer deps in this snippet!

@jarodabel Can you remove TS from dev deps and update the version in peer deps, please?

Reply:
and actually one more thing,  why is it this package  that is responisble for consistent TS version accross our apps ?

Reply:
Because this package has our standard prettier, tslint, etc. config, it's a good place to standardize our TS version as well.